src/build/__tests__/build-test.js

```diff
@@ -149,4 +168,16 @@ describe('build()', () => {
       Timestamp: { serialize },
     }, undefined, false)).forEach((type, i) => expectTypesEqual(type, [Record, Timestamp][i]));
   });
+
+  it('should allow directives configuration using __schema', () => {
+    const config = { __schema: { directives: [customDirective] } };
+    const target = build(schemaSource + querySourceWithDirective, config, undefined, false);
+    expectSchemasEqual(target, SchemaWithDirective);
+  });
+
+  it('should not build schema with default directives', () => {
```

Comment:
Hmm, it seems to me that the second test should be in `buildSchema-test.js`. And it would seem that the second test is the base case that's already tested in that file. The only new change in functionality is that `build` should know to pass the object in `__schema` to `buildSchema`.